onnxruntime/core/optimizer/matmul_bn_fusion.cc

```diff
@@ -0,0 +1,294 @@
+#include "core/optimizer/matmul_bn_fusion.h"
+#include "core/graph/graph_utils.h"
+#include "core/optimizer/initializer.h"
+#include "core/optimizer/utils.h"
+
+
+namespace onnxruntime
+{
+    void AddNodesToRemove(
+        Node::NodeConstIterator currItr,
```

Comment:
I propose `curr_iterator` (not a fan of abbreviations when it only saves a few letters). Also, don't forget this is in the core part of the ORT repo so we should follow the same indentation rules, braces placement and use snake_case.

Reply:
Good point. I will make those changes. I have moved this PR to draft and will change the status Ready to Review again.

Reply:
> should follow the same ... rules
⬆👍 "When in Rome, do as the Romans do."

> not a fan of abbreviations when it only saves a few letters
⬆👍 Or even clearer, `current_iterator`, since "ent" only saves a few letters. Write code with the future readers in mind, those poor souls who will maintain this code long after we are gone (for which English may be a secondary language, and thus abbreviations are unclear).